static/js/redux/ui/advising.jsx

```diff
@@ -33,9 +34,11 @@ class Advising extends React.Component {
       orientation: !mql.matches ? 'landscape' : 'portrait',
       selected_semester: null,
       transcript: null,
+      displayed_semesters: [ 'Fall 2021' ], // TODO: default to latest semester dynamically
```

Comment:
I would instead set this to null by default. Then, if this is null, display (instead of the dropdowns) a message like "Import SIS data to view your course history" or something?

Reply:
But shouldn't the latest semester always be shown from the original dashboard?

Reply:
hm true but i think we want students to import their sis data first and avoid being in a weird limbo stage where they are using the dashboard without importing their data

Reply:
so, i would say, make this null by default. then, when student's import their data, this semester will show up (if you do what i mentioned in the other comment)